app/assets/javascripts/discourse/app/controllers/group-messages.js

```diff
@@ -1,3 +1,10 @@
 import Controller from "@ember/controller";
+import { service } from "@ember/service";
 
-export default class GroupMessagesController extends Controller {}
+export default class GroupMessagesController extends Controller {
+  @service router;
+
+  get isGroup() {
+    return this.router.currentRoute.parent.name === "group.messages";
+  }
```

Comment:
This looks weird. Isn't this controller a group controller, so this should always be true?

In fact we can look up this controller anytime anywhere, even when it's not the active one, and that's the existing approach used on the user view.